src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -149,14 +149,12 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 		scriptBuilder.WriteString("if [ -d node_modules ] || [ -L node_modules ]; then\n")
 		// We move the directory/link first to prevent node from start using it
 		scriptBuilder.WriteString("    mv -f node_modules _del_node_modules || true\n")
-		scriptBuilder.WriteString("    nohup rm -fr _del_node_modules &> /dev/null &\n")
 		scriptBuilder.WriteString("fi\n\n")
 		scriptBuilder.WriteString("if [ -d /node_modules/.bin ]; then\n")
-		// We move the directory/link first to prevent node from start using it
-		scriptBuilder.WriteString("\n# creating symbolic links for local node_module's directory\n")
-		scriptBuilder.WriteString("ln -s /node_modules ./node_modules \n\n")
-		scriptBuilder.WriteString("\n# creating symbolic links for local node_module's bin directory\n")
-		scriptBuilder.WriteString("ln -s /node_modules/.bin ./node_modules/.bin \n\n")
+		// Create a symlink to extracted nodemodules directory so that binaries mentioned in package.json(ex: 'ng serve')
```

Comment:
porting fix from patch branch

Reply:
the mv with `-f` this case overrides the current `_del_node_modules` if it exists?

Reply:
Should we be worried about if it overrides as we anyway not going to use it?